sample/src/main/java/co/nimblehq/sample/ui/SurveyUiModel.kt

```diff
@@ -0,0 +1,6 @@
+package co.nimblehq.sample.ui
+
+data class SurveyUiModel<T> constructor(
+    val success: T? = null,
+    val error: Int? = null
```

Comment:
optimize later: better send back a custom exception/error that we may get the error type + error message at the same time, instead of just stringRes for now

Reply:
@sleepylee we should hav a ticket for the next phase, about error handling

Reply:
same here, we shall resolve the conversation with a ticket ref